tools/cli-bootstrap/.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.py -crlf
```

Comment:
We could consider just doing this:

###############################################################################
# Set default behavior to automatically normalize line endings.
###############################################################################
* text=auto

in a top level .gitattributes (that's what most other dotnet projects do).